app/models/controlled_work_document_content.rb

```diff
@@ -154,15 +164,24 @@ def combined_childcare_costs
       (session_data.dig("api_response", "assessment", "partner_disposable_income", "childcare_allowance") || 0)
   end
 
-  def additional_non_smod_properties_sum(attribute)
-    additional_non_smod_properties.sum { _1[attribute] }
+  def additional_properties_sum(attribute, *modifier)
+    additional_properties(*modifier).sum { _1[attribute] }
   end
 
-  def additional_non_smod_properties
-    @additional_non_smod_properties ||= [
-      (session_data.dig("api_response", "assessment", "capital", "capital_items", "properties", "additional_properties") unless additional_property_in_dispute?),
-      session_data.dig("api_response", "assessment", "partner_capital", "capital_items", "properties", "additional_properties"),
-    ].flatten.compact
+  def additional_properties(property_type = "all")
+    raise "Invalid modifier in YML" unless VALID_PROPERTY_TYPES.include?(property_type)
+
+    @additional_properties = if property_type == "non_smod"
```

Comment:
You could delete `@additional_properties =` entirely - the variable doesn't get read anywhere, so there's nothing gained by setting it here.